src/App.js

```diff
@@ -15,9 +19,16 @@ const App = () => {
   const { total, next } = state;
   const calc = { total, next };
   return (
-    <div className="App">
-      <Calculator handleClick={handleClick} calc={calc} />
-    </div>
+    <>
+      <BrowserRouter>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator handleClick={handleClick} calc={calc} />} />
+          <Route path="/quote" element={<MyQuotation />} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
- [ ] The `<Quotes>` component should be imported into the main component and rendered here. Currently, you're rendering the Quote component as a page and the requirement says it should be rendered below the calculator. Therefore, you are to call the Calculator component here and then place the Quote component below it like this: 👍🏽

```jsx
<>
     <Calculator />
      <Quotes />
</>
```